### PR TITLE
fixed some mistakes

### PR DESCRIPTION
**Pull request by gkwan-ibm**
- liveness -> readiness
- LMP version
- add a statement to clarify the port-forward command

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1441,7 +1441,7 @@ ReadinessCheck.java
 ----
 include::finish/module-health-checks/src/main/java/io/openliberty/deepdive/rest/health/ReadinessCheck.java[]
 ----
-The [hotspot=Readiness file=2]`@Readiness` annotation indicates that this class is a readiness health check procedure. Navigate to the http://localhost:9080/health/ready URL to check the status of the liveness health check. This tests the connection to the PostgreSQL container that was created earlier in the guide. If the connection is refused, a status of `DOWN` is returned.
+The [hotspot=Readiness file=2]`@Readiness` annotation indicates that this class is a readiness health check procedure. Navigate to the http://localhost:9080/health/ready URL to check the status of the readiness health check. This tests the connection to the PostgreSQL container that was created earlier in the guide. If the connection is refused, a status of `DOWN` is returned.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
The [hotspot=Readiness file=2]`@Readiness` annotation indicates that this class is a readiness health check procedure. Navigate to the http://localhost:9080/health/ready URL to check the status of the readiness health check. This readiness check tests the connection to the PostgreSQL container that was created earlier in the guide. If the connection is refused, a status of `DOWN` is returned.
```

File: README.adoc [hunk 2]
```diff
@@ -2311,6 +2311,8 @@ Run the following command to set up port forwarding to access the `inventory` mi
 kubectl port-forward svc/inventory-deployment 9443
 ```
 
+The `port-forward` command should hold up the command-line session.
```
Comment (dmuelle) [inline README.adoc]: what does "hold up" mean in this context. Also, we should avoid using "should" to refer to a function as it implies uncertainty.

Comment (gkwan-ibm) [reply README.adoc]: - the word "should" can be removed
- "hold up" means that the command session cannot be typed in anything more. Any better word?

Comment (dmuelle) [inline README.adoc]:
```suggestion
The `port-forward` command pauses the command-line session until you click Ctrl+C.
```

File: README.adoc [hunk 3]
```diff
@@ -2311,6 +2311,8 @@ Run the following command to set up port forwarding to access the `inventory` mi
...
+
 ifndef::cloud-hosted[]
 The application might take some time to get ready. See the https://localhost:9443/health URL to confirm that the `inventory` microservice is up and running. You can now check out the service at the https://localhost:9443/openapi/ui/[https://localhost:9443/openapi/ui/^] URL. The servers dropdown list shows the `\https://localhost:9443/dev` URL. Or, you can run the following command to access the inventory microservice:
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
The application might take some time to get ready. See the https://localhost:9443/health URL to confirm that the `inventory` microservice is up and running. You can now check out the service at the https://localhost:9443/openapi/ui/[https://localhost:9443/openapi/ui/^] URL. The server's dropdown list shows the `\https://localhost:9443/dev` URL. Or, you can run the following command to access the inventory microservice:
```

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Pull request merged**